Content.Server/Speech/Components/ListenWireAction.cs

```diff
@@ -72,46 +81,13 @@ public override void Pulse(EntityUid user, Wire wire)
         if (!GetValue(wire.Owner) || !IsPowered(wire.Owner))
             return;
 
-        // We have to use a valid euid in the ListenEvent. The user seems
-        // like a sensible choice, but we need to mask their name.
-
-        // Save the user's existing voicemask if they have one
-        var oldEnabled = true;
-        var oldVoiceName = Loc.GetString("wire-listen-pulse-error-name");
-        string? oldSpeechVerb = null;
-        if (EntityManager.TryGetComponent<VoiceMaskComponent>(user, out var oldMask))
-        {
-            oldEnabled = oldMask.Enabled;
-            oldVoiceName = oldMask.VoiceName;
-            oldSpeechVerb = oldMask.SpeechVerb;
-        }
-
-        // Give the user a temporary voicemask component
-        var mask = EntityManager.EnsureComponent<VoiceMaskComponent>(user);
-        mask.Enabled = true;
-        mask.VoiceName = Loc.GetString("wire-listen-pulse-identifier");
-        mask.SpeechVerb = SpeechVerb;
-
         var chars = Loc.GetString("wire-listen-pulse-characters").ToCharArray();
         var noiseMsg = _chat.BuildGibberishString(chars, NoiseLength);
 
-        var attemptEv = new ListenAttemptEvent(wire.Owner);
-        EntityManager.EventBus.RaiseLocalEvent(wire.Owner, attemptEv);
-        if (!attemptEv.Cancelled)
-        {
-            var ev = new ListenEvent(noiseMsg, user);
-            EntityManager.EventBus.RaiseLocalEvent(wire.Owner, ev);
-        }
+        if (!EntityManager.TryGetComponent<RadioMicrophoneComponent>(wire.Owner, out var radioMicroPhoneComp))
+            return;
 
-        // Remove the voicemask component, or set it back to what it was before
-        if (oldMask == null)
-            EntityManager.RemoveComponent(user, mask);
-        else
-        {
-            mask.Enabled = oldEnabled;
-            mask.VoiceName = oldVoiceName;
-            mask.SpeechVerb = oldSpeechVerb;
-        }
+        _radio.SendRadioMessage(wire.Owner, noiseMsg, _protoMan.Index<RadioChannelPrototype>(radioMicroPhoneComp.BroadcastChannel), wire.Owner);
```

Comment:
This part is not identical anymore. Before if pulsing the wire on the intercom it had "electricity" as the speaker, but now it is the intercom itself.
![grafik](https://github.com/user-attachments/assets/ab6714bd-bfd3-4d64-b394-efadc58408e8)